src/Tribe/Aggregator/Cron.php

```diff
@@ -465,6 +442,42 @@ public function start_batch_pushing_records() {
 		}
 	}
 
+	/**
+	 * Get the first set of pending schedule records to be processed for batch pushing.
+	 *
+	 * @since TBD
+	 *
+	 * @return WP_Query The result of the Query.
+	 */
+	private function get_batch_pushing_records() {
+		return Records::instance()->query(
+			[
+				'post_status'    => Records::$status->pending,
+				'posts_per_page' => 250,
```

Comment:
:warning: Having more than 100 posts returned per page may lead to severe performance problems.

<sub>hash: 7da166fe33d427a1d5c3c83c98b5de2b</sub>